x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -44,7 +44,10 @@ export type Event = AlertEvent | SafeEndpointEvent;
  */
 export const ANCESTRY_LIMIT: number = 2;
 
-interface EventOptions {
+/**
+ * The configuration options for generating an event.
+ */
+export interface EventOptions {
```

Comment:
Thoughts on moving this to a common `types` module? In my opinion that's worked pretty well so far. (if its not working well, I'd like to discuss that too.)

Reply:
Good idea, will do.